src/services/EmissionEstimatesService.ts

```diff
@@ -548,11 +554,16 @@ export abstract class EmissionEstimatesService {
         id: string,
         data: {
             shipment: Shipment
-            legs: Array<{
-                route: ShippingRoute
-                method: ShippingMethod
-                countryCode?: ShippingCountryCode
-            }>
+            legs: Array<
+                | {
```

Comment:
`| {` does this work?

Reply:
🤔 Didn't notice this. The build succeeds so it does result in 'something'. Let me test locally to see how is this handled.

Reply:
@rbruggem This is actually made like this by the prettify step, it's not there before 👀

So to your question, yes this is working:
<img width="593" alt="Screenshot 2023-06-06 at 11 30 06" src="https://github.com/lune-climate/lune-ts/assets/3910179/97871480-a21a-44ea-9b60-4f86250a2324">

Reply:
okay!